pkg/kubecli/install.go

```diff
@@ -0,0 +1,157 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ * Copyright 2018 The Kubernetes Authors.
```

Comment:
just curious, why do we use both "Red Hat" and "Kubernetes Authors" everywhere?

Reply:
I am not sure just copy it from another file in the kubecli folder.
@fabiand Can you help us with this?
What is the right Copyright.

Reply:
kubecli was created by @rmohr and @cynepco3hahue 

Can you help?

Reply:
I do not have any idea :smile: Maybe we use some code from Kubernetes, but I think it not related to this file.